tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>better specify that "traditional CPU systems" means a supercomputer, otherwise it seems it is also required on a laptop.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Well, if the user has an AMD Ryzen workstation with 16 cores (for example), then they probably want to use mpi4py if they are doing something more with toast than just running a notebook with a few samples.  I will definitely clarify though.  I have started an "intro_parallel.ipynb" where I am going to discuss using IPython.parallel with mpi4py.  I'll reference that in the serial notebook.</p>

Reply:
<p>Tried to clarify that toast parallelism is mainly through MPI, so that any system with more than a few cores will benefit from having mpi4py installed.</p>